### PR TITLE
add mysql ports

### PR DESCRIPTION
**Pull request by joonmyung**
add mysql ports for mysql client tool connection.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ services:
     volumes:
       - mysql:/var/lib/mysql
       # - /etc/localtime:/etc/localtime:ro
-
+    ports:
+      - ${PRISMA_DB_PORT:-3306}:${PRISMA_DB_PORT:-3306}
```
[the remark this answers is not in the document]
Comment (joonmyung) [reply docker-compose.yml]: for more flexibility, we can add docker external port param. like this.

      - ${EXTERNAL_DB_PORT:-3306}:${PRISMA_DB_PORT:-3306}

this is what you thought then I will PR again.

[the remark this answers is not in the document]
Comment (hyochan) [reply docker-compose.yml]: I think this is more likely in a reverse way(correct me if I am wrong).
```
  - ${LOCAL_MYSQL_PORT:-3306}:${DOCKER_MYSQL_PORT:-3306}
```

Also, providing this to readme would be required.

[the remark this answers is not in the document]
Comment (joonmyung) [reply docker-compose.yml]: PRISMA_DB_PORT and DOCKER_MYSQL_PORT have to same for communicate each other.
so, I don't think DOCKER_MSSQL_PORT is needed.

[the remark this answers is not in the document]
Comment (hyochan) [reply docker-compose.yml]: Alright. If there is some duplicate, then we should prevent that.

**Comment by cometkim** [comment]
@joonmyung Can you share your case with additional DB tools?

**Comment by joonmyung** [comment]
> @joonmyung Can you share your case with additional DB tools?
In my case, I can't connect docker mysql without ports params.
(I used Sequel Pro )

please refer below link.
https://github.com/prisma/prisma/issues/1125

**Comment by hyochan** [comment]
As we said in `meetup`, let's just handle this personally like below (there may be other ways around).

1. Find port of `mysql` in docker.
```
$ docker port mysql 3306
0.0.0.0:32768
```

2. Find your docker machine IP
```
$ docker-machine ip
192.168.99.100
```

3. Connect to `mysql` with a host of `192.168.99.100:32768`.

**Comment by geoseong** [comment]
@hyochan Did you face the problem when you start `npm run docker:up`?

```
Creating network "talktalk-node_default" with the default driver
Creating talktalk-node_mysql_1  ... error
Creating talktalk-node_prisma_1 ... 

ERROR: for talktalk-node_mysql_1  Cannot start service mysql: driver failed programming external connectivity on endpoint talktalk-node_mysql_1 (54e73ec036f52400fc6ba4fdf1d75470d4a8fe037f2a6a66eae8b6947e6b7a86): Error starting userland proxy: Bind for 0.0.0.0:3306 failed:Creating talktalk-node_prisma_1 ... done

ERROR: for mysql  Cannot start service mysql: driver failed programming external connectivity on endpoint talktalk-node_mysql_1 (54e73ec036f52400fc6ba4fdf1d75470d4a8fe037f2a6a66eae8b6947e6b7a86): Error starting userland proxy: Bind for 0.0.0.0:3306 failed: port is already allocated
```

**Comment by hyochan** [comment]
@geoseong I think that is another issue. Logs tell you that there is already something listening to 3306 port.

**Comment by geoseong** [comment]
@hyochan Yeah. This was happening because I hadn't know how to turn off MySQL Server on my computer... I just turned off in my Control Panel, and run `talktalk-node`'s `npm run docker:up`. so everything is working well. haha 😅

**Pull request closed without merging**